### DISCUSSION TITLE
content adaptive

### DISCUSSION THREAD

**Discussion by zx405** [bcakmakoglu/vue-flow#1438] [Q&A] [answered]
How can I make the content of my inside adapt to the changes in the size of the browser window? I see that the official website seems to achieve this function

**Comment by bcakmakoglu** [comment, marked as answer]
[You can find the implementation from the docs right here.](https://github.com/bcakmakoglu/vue-flow/blob/master/docs/components/home/flows/Intro.vue)

Although it's quite old and can probably be cleaned up a bit 😄
